diff --git a/configure b/configure
index 93c32cc..669299d 100755
--- a/configure
+++ b/configure
@@ -29,7 +29,7 @@ if [ "$INCLUDE_DIR" ] || [ "$LIB_DIR" ]; then
   echo "Found INCLUDE_DIR and/or LIB_DIR!"
   PKG_CFLAGS="-I$INCLUDE_DIR $PKG_CFLAGS"
   PKG_LIBS="-L$LIB_DIR $PKG_LIBS"
-elif [ "$PLATFORM" = "Darwin" ] && [ "$DISABLE_AUTOBREW" != 1 ]; then
+elif [ "$PLATFORM" = "Darwin" ]; then
   brew --version 2>/dev/null
   if [ $? -eq 0 ]; then
     BREWDIR=`brew --prefix`
